modules/bot/controllers/privates/AdminVoteBanController.php

```diff
@@ -103,4 +105,50 @@ public function actionUpdate($chatId = null)
 
         return $this->actionIndex($chatId);
     }
+
+    public function actionEnterLimit($chatId = null)
+    {
+        $this->getState()->setName(self::createRoute('update-limit', [
+            'chatId' => $chatId,
+        ]));
+        return ResponseBuilder::fromUpdate($this->getUpdate())
+            ->sendMessage(
```

Comment:
До изменения вы использовали ведь editMessageTextCommand)
```suggestion
            ->editMessageTextOrSendMessage(
```

Reply:
да, что-то мне показалось , что неправильная логика, я там и кнопку удалил, сейчас переделаю